orchagent/orchdaemon.cpp

```diff
@@ -513,6 +516,17 @@ void OrchDaemon::start()
                 }
             }
         }
+        if (gIsSwitchShutdown) {
```

Comment:
in normal executing, does the code reach here? i saw if select timeout, it just go to continue.

Reply:
Very good catch. I just found out, that this notification comes in a different thread and hence all the selectable orchs, could be idling causing timeout.

I just moved it up in the loop.